index.html

```diff
@@ -1338,6 +1338,19 @@ <h2>
 key formats for RSA MUST cause the DID Document Processor to raise an error.
             </td>
           </tr>
+          <tr>
+            <td>
+secp256k1
+            </td>
+            <td>
+Secp256k1 key values MAY be expressed as the raw 32-byte value encoded in
+base58-btc format using the <code>publicKeyBase58</code> property or
+MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
```

Comment:
Is it common for anyone to use JWK for this key type? If not, we could just specify the one format here.

Reply:
Yes, use of JWKs for Ed25519 is common.  That's why members of the community developed RFC 8037 - to define how to use the 25519 and 448 curves with JSON.

Reply:
> Yes, there are people using the secp256k1 JWK representation

Could you get these people to fill out the survey: 

https://forms.gle/Hovf3irBJ5KwgXLQ6

If any of these people are writing base cryptography libraries (not wrapping libraries), please also get them to fill out:

https://docs.google.com/spreadsheets/d/12dwUaAruKKpq3a3IfEMEMhpRhI7oM1tQnkmDbW2VGoU/edit